src/js/components/RequestErrorMsg.js

```diff
@@ -46,7 +46,7 @@ class RequestErrorMsg extends React.Component {
 
 RequestErrorMsg.defaultProps = {
   columnClasses: {},
-  header: "Cannot Connect With The Server",
+  header: "There was a problem with network request",
```

Comment:
Not sure how important this is, but I'd avoid the word "Problem". 
What about something like "DC/OS UI has difficulties loading the requested information, please try again"? 🤔

Reply:
How about: "DC/OS UI cannot retrieve the requested information at this moment."

Reply:
I like it!